prov/psm2/src/psmx2.h

```diff
@@ -651,8 +651,8 @@ struct psmx2_fid_cntr {
 	struct psmx2_fid_domain	*domain;
 	int			events;
 	uint64_t		flags;
-	atomic_t		counter;
-	atomic_t		error_counter;
+	atomic32_t		counter;
+	atomic32_t		error_counter;
```

Comment:
Thank you! Please, find it updated. Btw, as I can see, there are some places where signed (these counters) and unsigned values (thresholds - size_t and uint64_t), but it isn't a problem while counter exceeds no MAX_INT64 value, just a root of warnings. Should it be fixed in this PR? In psm the same